src/core/Table/Table.tsx

```diff
@@ -213,6 +213,11 @@ export type TableProps<
    * @default false
    */
   isResizable?: boolean;
+  /**
+   * Use to have zebra-striped table rows.
+   * @default false
+   */
+  isStriped?: boolean;
```

Comment:
Maybe something like `styleType` : default or striped would be better? This way we could add more styles later.

Reply:
Changed prop and added separate story

Reply:
Might be better to do `zebra-rows`. we could add zebra columns in the future